### PR TITLE
aarch + ppc64le again for 0.22

### PR DESCRIPTION
**Pull request by hmaarrfk**
Checklist
* [ ] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by hmaarrfk** [comment]
This is **very** likely ready to go. Unfortunately, I have to recommend skipping the tests for PPC64LE and AARCH64. The build time is **quite** variable. Sometimes, the CIs can run through the whole build + test procedure in 23 minutes, other times, it timesout after only 10% of the tests. This doesn't seem related to the test suite, since it won't even finish the build phase until 28 mins...

**Comment by hmaarrfk** [comment]
See odd timing failure here https://cloud.drone.io/conda-forge/scikit-learn-feedstock/36/2/2

**Comment by hmaarrfk** [comment]
finally:
image


the variance on the compilation + test times is insane.

**Comment by ogrisel** [comment]
> the variance on the compilation + test times is insane.

Do you have any idea why? Noisy neighbors using the same hardware in concurrently running containers without proper cgroup quotas for CPU / disk / RAM resources maybe?

**Comment by hmaarrfk** [comment]
Not too sure. We can probably ping drone.

But for now. It has been passing consistently. Would you like to merge?

I the remaining failures are all legitimately issues that I reported.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,12 +10,15 @@ source:
...
+  number: 1
   skip: True  # [py2k]
   script:
     - rm $PREFIX/include/cblas.h  # [not win]
     - del /f %LIBRARY_INC%\cblas.h  # [win]
-    - "{{ PYTHON }} -m pip install . --no-deps -vv"
+    # build the C files in parallel. This helps beat the
+    # time limit consistently on drone.io
+    - {{ PYTHON }} setup.py build_ext -j{{ CPU_COUNT }}
+    - {{ PYTHON }} -m pip install . -vv
```
Comment (rth) [inline recipe/meta.yaml]: Do we want to keep `--no-deps`?

Comment (hmaarrfk) [reply recipe/meta.yaml]: I'm 90 percent sure conda forge does this in their pip config

Comment (hmaarrfk) [reply recipe/meta.yaml]: A while back, I went on this hunt, and found that they isolate the environment:
https://github.com/scikit-image/scikit-image/issues/4261#issuecomment-557322707

Comment (jakirkham) [reply recipe/meta.yaml]: Actually [it's done in conda-build]( https://github.com/conda/conda-build/blob/1c76172835eec45ebaf5a0fa17dace84dc4e3475/conda_build/build.py#L2368 ), but yeah we don't need it here.

Comment (hmaarrfk) [reply recipe/meta.yaml]: https://github.com/conda/conda-build/pull/3271/files

Comment (hmaarrfk) [reply recipe/meta.yaml]: https://github.com/conda/conda-build/blob/master/CHANGELOG.txt#L577

Comment (rth) [reply recipe/meta.yaml]: Thanks for the confirmation!

**Pull request merged**